src/styles/withTheme.js

```diff
@@ -0,0 +1,33 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import wrapDisplayName from 'recompose/wrapDisplayName';
```

Comment:
I tried using `withHelper`, but from what I can see https://github.com/acdlite/recompose/blob/master/src/packages/recompose/createHelper.js#L18-L24 it would then require users to call the HOC like before, i.e. `withTheme()(BaseComponent)`, which was one of the things we wanted to avoid.